clients/cli/src/keys.rs

```diff
@@ -24,21 +79,22 @@ mod tests {
 
     #[test]
     fn valid_checksum_address() {
+        // This address has correct EIP-55 checksum
         assert!(is_valid_eth_address(
             "0x52908400098527886E0F7030069857D2E4169EE7"
```

Comment:
I'm seeing all uppercase characters, which needs to be mixed in order to do checksum validation. Since you have checksum tests below, you can probably remove this one.